src/app/onboarding/components/glam/archive-type-select/archive-type-select.component.scss

```
@@ -1,3 +1,4 @@
+/*N @format */
```

```suggestion
/* @format */
```
The prettier pragma got a bit messed up here.